meta-mender-core/recipes-bsp/u-boot/u-boot-mender-common.inc

```diff
@@ -81,9 +81,18 @@ do_provide_mender_defines() {
             ;;
     esac
 
-    MENDER_DTB_NAME=$(echo "${KERNEL_DEVICETREE}" | grep -o '[^ ]*$')
-    if [ "$MENDER_DTB_NAME" != "${KERNEL_DEVICETREE}" ]; then
-        bbwarn "More than one dtb file specified in KERNEL_DEVICETREE. Only one should be specified. Choosing the last one."
+    MENDER_DTB_NAME=$(echo "${KERNEL_DEVICETREE}" | sed 's/\s/\n/g' | grep -v '.dtbo' | grep -o '[^ ]*$')
```

Comment:
I suggest using `sed -r 's/\s+/\n/g'`, then I think you can drop the final `grep -o`. That one is for getting rid of empty lines, right?